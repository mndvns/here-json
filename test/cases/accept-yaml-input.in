./bin/here-json -p <<EOF
---
  this: is
  some: yaml-y input
EOF