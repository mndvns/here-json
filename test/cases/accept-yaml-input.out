{
  "this": "is",
  "some": "yaml-y input"
}